src/App.js

```diff
@@ -1,26 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
Your app does not follow the design. Please make it exact same as in the given design.
Yours:
![yours](https://user-images.githubusercontent.com/35504149/158440283-f60e3189-3291-49f7-8067-751beb985914.png)
Original design:
https://gitlab.com/microverse/guides/tse/code_review/code_review_guidelines_2_0/-/raw/master/react_redux/math_magicians/images/calculator.png